### ISSUE TITLE
Error on installing specific version of Postgresql

### ISSUE THREAD

**Issue by ahoora08** [Homebrew/linuxbrew-core#17843]
Hi,

When I try to install Postgresql 11 using the command: brew install postgresql@11 on Ubuntu 18.04, I got the following error:
```
Installing postgresql@11
==> Downloading https://ftp.postgresql.org/pub/source/v11.6/postgresql-11.6.tar.bz2
Error: An exception occurred within a child process:
  NoMethodError: undefined method `sdk_path' for OS::Mac:Module
```
It used to work but since setting the Postgresql 12 as a stable version, it doesn't work anymore!

**Comment by issyl0** [comment]
Thanks for making us aware. A fix is being worked on in #17810.

**Comment by iMichka** [comment]
Fixed by #17810
